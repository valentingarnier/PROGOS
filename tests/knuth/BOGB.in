4 K
1 1
1 2
2 1
4 0
