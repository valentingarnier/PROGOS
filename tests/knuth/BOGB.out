What size (2, 3, 4, 5)? What strategy ([B]rute force, B[i]tfield, [K]nuth)? Y Y B B
Please score attempt (positions, colors): R G B B
Please score attempt (positions, colors): P B G B
Please score attempt (positions, colors): B O G B
Please score attempt (positions, colors): Found solution: B O G B
